src/components/IssueLabels.js

```diff
@@ -0,0 +1,29 @@
+import { array, number } from 'prop-types'
+import React from 'react'
+import { spacing } from '../theme'
+import Flex from './Flex'
+import IssueLabel from './IssueLabel'
+
+function IssueLabels({ labels, totalCount }) {
+  const remainingLabels = totalCount - labels.length
```

Comment:
Is there better name for this variable? I don't think that it's obvious that `remainingLabels` is a number.

Reply:
`remainingCount`?
You have ideas?

Reply:
`remainingLabelCount` is kind of verbose, but maybe fine.